spring-boot-starter-camunda-sdk/CONTRIBUTORS.md

```diff
@@ -59,4 +59,4 @@
 * [minglei.wen](https://github.com/wenminglei)
 * [pcln-jaynil-patel](https://github.com/pcln-jaynil-patel)
 * [rob2universe](https://github.com/rob2universe)
-
+* [hem-verma](https://github.com/hem-verma)
```

Comment:
Hey @jonathanlukas, the Java Checkstyle CI is complaining about this file, can you add an empty line at the end ?

Reply:
There is an empty line at the end.

Reply:
I let the formatter run. It somehow requires 2 empty lines...